todo_app/templates/index.html

```diff
@@ -7,11 +7,54 @@ <h1 class="display-4">To-Do App</h1>
     <p class="lead">Just another to-do app.</p>
   </div>
 
+  <!-- Form -->
+  <h2>New item</h2>
+  <form action="{{ url_for('add_new_item') }}" method="post">
+    <label for="item">Enter your new item: </label>
+    <input type="text" name="new_item" id="item" required>
+    <input type="submit" value="Add new item">
+  </form>
+
+  <!-- Display list -->
   <div class="row">
     <div class="col-md-12">
       <h2>Items</h2>
       <ul class="list-group mb-4">
-        <!-- List the todo items here -->
+        <table class="table">
+          <thead>
+            <tr>
+              <th class="task-id">#</th>
+              <th class="task">Task Name</th>
+              <th class="status">Status</th>
+              <th class="completed">Completed?</th>
+              <th class="remove">Remove</th>
+
+
+            </tr>
+          </thead>
+          {% for item in todo_items %}
+            <tbody>
+              <td>{{ item.get('id') }}</td>
```

Comment:
This is really nitpicky so feel free to disregard it, but:
These are database ids, not just indices. I would probably avoid showing them in the frontend because users don't need them for anything, and it will be nicer to hide this implementation detail from them if possible.